src/RestApi/Controllers/ProductCategories.php

```diff
@@ -115,6 +133,7 @@ public function prepare_item_for_response( $item, $request ) {
 			'description' => $item->description,
 			'image'       => null,
 			'permalink'   => get_term_link( $item->term_id, 'product_cat' ),
+			'reviews'     => $reviews_count,
```

Comment:
I think being more explicit about what this represents would be a good idea. We use `review_count` on the product endpoint.